### DISCUSSION TITLE
Italian Language Leader

### DISCUSSION THREAD

**Discussion by luca-angemi** [OHF-Voice/intents#429] [Language Leaders]
Hi,

I'd like to apply. I'm Italian and fluently speak and write English and Spanish (Spain and Chile variations).

In any case, I'm happy to help with translations. 

Cheers
Luca

**Comment by synesthesiam** [comment]
@auanasgheps and @xraver are current language leaders. Please coordinate with them, we'd be happy for you to contribute :slightly_smiling_face:
